apm-agent-plugins/apm-opentracing-plugin/src/main/java/co/elastic/apm/agent/opentracing/impl/ApmSpanInstrumentation.java

```diff
@@ -276,6 +277,15 @@ private static boolean handleSpecialSpanTag(Span span, String key, Object value)
                     span.withType("ext");
                 }
                 return true;
+            } else if ("http.status_code".equals(key) && value instanceof Number) {
```

Comment:
this was the missing part in our opentracing bridge implementation (we only did this for transactions, not spans).